Source/Core/Core/Debugger/RSO.cpp

```diff
@@ -380,14 +379,21 @@ void RSOView::Apply(PPCSymbolDB* symbol_db) const
       Symbol* symbol = symbol_db->AddFunction(address);
       if (!symbol)
         symbol = symbol_db->GetSymbolFromAddr(address);
+
+      const std::string export_name = GetExportName(rso_export);
       if (symbol)
       {
-        symbol->name = GetExportName(rso_export);
-        count += 1;
+        // Function symbol
+        symbol->name = export_name;
+      }
+      else
+      {
+        // Data symbol
+        symbol_db->AddKnownSymbol(address, 0, export_name, Symbol::Type::Data);
```

Comment:
Ah okay. Might be best to leave that for another PR then, when we figure out how the hash is generated.